ngraph/core/include/ngraph/function.hpp

```diff
@@ -52,6 +53,16 @@ namespace ngraph
                  const ParameterVector& parameters,
                  const std::string& name = "");
 
+        Function(const ResultVector& results,
```

Comment:
Could you use the new API in ie_ir_reader.cpp for Assigns and check Kaldi e2e models? This will be proof of the functionality of the feature.

Reply:
done, precommit Kaldi tests passed https://openvino-ci.intel.com/job/private-ci/job/ie/job/e2e-tests-linux-ubuntu18-e2e/20120/e2e-tests-gna/